.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -10,7 +10,7 @@ body:
         Before you do, please ensure you are filing the issue in the right
         place. 
 
-        * If you have a question or if an issue is not right for what you mean
+        If you have a question or if an issue is not right for what you mean
```

```suggestion
        If you have a question or if an issue is not right for what you mean
```
The wording here is a bit awkward. Maybe something like "If you have a question, or would like to start a discussion that's not does not fit the issue criteria, see (link)"